README.md

```diff
@@ -26,9 +206,109 @@ We also use bundling techniques to optimize code delivery and reduce load times
+
+The following is a list of packages included in the Unified SDK:
+
+### Blockchain Data
+
+The Blockchain Data SDK provides a set of wrapper functions for interacting with the Immutable Blockhain Data APIs. It provides a simple and easy-to-use interface for querying blockchain data, such as assets, orders, and trades. 
+
+The Blockchain Data SDK package is built using TypeScript and Rollup. The package is exposed to the SDK package that is published to NPM.
+
+More information can be found in the README file in the Blockchain Data SDK folder [here](/packages//blockchain-data//sdk//README.md).
+
+### Checkout
+
+The Checkout SDK folder provides both functions and user interfaces for interacting with various wallets and L2 networks. It provides all the tools to enable transaction ready gamers including on ramping, dex swaps, bridging and more.
+
+The folder contains the Checkout SDK package and the Widgets Library package. The Checkout SDK package provides the core functionality for interacting with wallets and L2 networks, while the Widgets Library package provides a set of pre-built UI components that can be used to build a custom checkout experience.
+
+The Checkout SDK package is built using TypeScript and Rollup, while the Widgets Library package is built using TypeScript and React. The Checkout SDK and Widgets Library packages are exposed to the SDK package that is published to NPM.
+
+Sample apps are also included in the Checkout SDK folder to demonstrate how to use the SDK and Widgets Library packages. These sample apps can be run in development mode to test changes and new features before deploying them to production.
+
+More information can be found in the README file in the Checkout SDK folder [here](/packages//checkout//README.md).
+
+### Config
+
+The Config package exposes a configuration class and some utilities to create a configuration object. This object contains information used for the SDK setup by consumers. It ensures that all modules and packages within the SDK can access the necessary configuration settings in a consistent and reliable manner, as well as providing customers with a simple way to configure the SDK for their specific environment.
+
+The Config package is built using TypeScript and Rollup. The package is exposed to the SDK package that is published to NPM.
+
+### Game Bridge
+
+The Game Bridge SDK package allows the Game SDKs (Unity and Unreal) to communicate with the TypeScript SDK. The Game SDKs are used to build games that interact with the Immutable platform. This enables communication between the Game and Typescript SDKs, allowing the Game SDKs to leverage the existing TypeScript SDK.
+
+The TS SDK and game-bridge are bundled into an index file (HTML for Unity, JS for Unreal), which is how the Game SDKs use them.
```

Comment:
Unified SDK
SDK
TS SDK...
i suggest swap all of them to "Typescript SDK" for consistency

Reply:
Updated 💪